src/entries/popup/pages/home/NFTs/NFTDetails.tsx

```diff
@@ -804,9 +805,11 @@ const NFTAccordionAboutSection = ({
             </Inline>
             <Inline alignVertical="center" space="6px">
               <ChainBadge
-                chainId={chainIdFromChainName(
-                  nft?.network || ChainName.mainnet,
-                )}
+                chainId={
+                  chainNameToIdMapping[
+                    (nft?.network as ChainName) || ChainName.mainnet
+                  ]
+                }
```

Comment:
I assume doing `as` type assertion won't be an issue here right ?

Reply:
yeah but now that you mention i'll take another look to return that type from the nft interface to make it cleaner

Reply:
ok, this wasn't needed at all lol i updated